Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp

```diff
@@ -1271,14 +1271,29 @@ void Jit64::divwux(UGeckoInstruction inst)
           RCX64Reg Rd = gpr.Bind(d, RCMode::Write);
           RegCache::Realize(Ra, Rd);
 
-          if (d == a)
+          magic++;
+
+          // Use smallest magic number and shift amount possible
+          while ((magic & 1) == 0)
```

Comment:
`&& shift > 0`

Reply:
Don't think there is any risk of this being an infinite loop since divisions by powers of 2 are already handled elsewhere, but let's err on the side of caution.